packages/autorest.typescript/test/integration/generated/azureParameterGrouping/src/models/index.ts

```diff
@@ -33,7 +33,7 @@ export interface ParameterGroupingPostOptionalParameters {
 /** Parameter group */
 export interface ParameterGroupingPostReservedWordsParameters {
   /** 'from' is a reserved word. Pass in 'bob' to pass. */
-  from?: string;
+  fromParam?: string;
```

Comment:
Can we adjust the reserved words to make from not reserved when it is a property name?

Reply:
Also function, should be fine as a property name but not as an interface identifier

Reply:
Yeah, it seems introducing un-necessary breakings here. Can we mitigate this?